src/components/Proposal/Create/PluginForms/CreateTokenTradeProposal.tsx

```diff
@@ -0,0 +1,325 @@
+                      />
+                    </div>
+                    <div className={css.select}>
+                      <Field
+                        id="sendTokenAddress"
+                        name="sendTokenAddress"
+                        component={SelectField}
+                        options={Object.keys(supportedTokens()).map((tokenAddress) => {
+                          const token = supportedTokens()[tokenAddress];
+                          return { value: tokenAddress, label: token["symbol"] };
+                        })}
+                      />
+                    </div>
+                  </div>
+                </div>
+
+                <div className={css.reward}>
+                  <label htmlFor="receiveTokenAmountInput">
+                    Receive token amount
+                    <ErrorMessage name="receiveTokenAmount">{(msg) => <span className={css.errorMessage}>{msg}</span>}</ErrorMessage>
+                  </label>
+                  <div className={css.externalTokenInput}>
+                    <div className={css.amount}>
+                      <Field
+                        id="receiveTokenAmountInput"
+                        placeholder={"How many tokens to receive"}
+                        name="receiveTokenAmount"
+                        type="number"
+                        className={touched.receiveTokenAmount && errors.receiveTokenAmount ? css.error : null}
+                        min={0}
+                        step={1}
+                      />
+                    </div>
+                    <div className={css.select}>
+                      <Field
+                        id="receiveTokenAddress"
+                        name="receiveTokenAddress"
+                        component={SelectField}
+                        options={Object.keys(supportedTokens()).map((tokenAddress) => {
+                          const token = supportedTokens()[tokenAddress];
+                          return { value: tokenAddress, label: token["symbol"] };
+                        })}
+                      />
+                    </div>
+                  </div>
+                </div>
+
+                <div className={css.createProposalActions}>
+                  <TrainingTooltip overlay="Export proposal" placement="top">
+                    <button id="export-proposal" className={css.exportProposal} type="button" onClick={() => this.exportFormValues(values)}>
+                      <img src="/assets/images/Icon/share-blue.svg" />
+                    </button>
+                  </TrainingTooltip>
+                  <button className={css.exitProposalCreation} type="button" onClick={handleClose}>
+                      Cancel
+                  </button>
+                  <TrainingTooltip overlay="Once the proposal is submitted it cannot be edited or deleted" placement="top">
+                    <button className={css.submitProposal} type="submit" disabled={isSubmitting}>
```

Comment:
You'll have a Cleat Input Fields button here once you incorporate use of `IFormModalService`